neuromation/api/jobs.py

```diff
@@ -259,26 +261,85 @@ async def run(
         life_span: Optional[float] = None,
     ) -> JobDescription:
         url = self._config.api_url / "jobs"
-        payload: Dict[str, Any] = {
-            "container": _container_to_api(container, self._config),
-            "is_preemptible": is_preemptible,
-            "pass_config": pass_config,
-        }
-        if name:
-            payload["name"] = name
-        if tags:
-            payload["tags"] = tags
-        if description:
-            payload["description"] = description
-        if schedule_timeout:
-            payload["schedule_timeout"] = schedule_timeout
-        if restart_policy != JobRestartPolicy.NEVER:
-            payload["restart_policy"] = str(restart_policy)
-        if life_span is not None:
-            payload["max_run_time_minutes"] = int(life_span // 60)
-        if wait_for_jobs_quota:
-            payload["wait_for_jobs_quota"] = wait_for_jobs_quota
-        payload["cluster_name"] = self._config.cluster_name
+        payload = _job_to_api(
+            config=self._config,
+            image=container.image,
+            entrypoint=container.entrypoint,
+            command=container.command,
+            working_dir=container.working_dir,
+            http=container.http,
+            env=container.env,
+            volumes=container.volumes,
+            secret_env=container.secret_env,
+            secret_files=container.secret_files,
+            disk_volumes=container.disk_volumes,
+            tty=container.tty,
+            name=name,
+            tags=tags,
+            description=description,
+            pass_config=pass_config,
+            wait_for_jobs_quota=wait_for_jobs_quota,
+            schedule_timeout=schedule_timeout,
+            restart_policy=restart_policy,
+            life_span=life_span,
+        )
+        payload["container"]["resources"] = _resources_to_api(container.resources)
+        payload["is_preemptible"] = is_preemptible
+        auth = await self._config._api_auth()
+        async with self._core.request("POST", url, json=payload, auth=auth) as resp:
+            res = await resp.json()
+            return _job_description_from_api(res, self._parse)
+
+    async def run_from_preset(
```

Comment:
This api is intended to be the main way to start a job.  
`run_from_preset()` is a cumbersome name.
Please use something shorter and easier to remember, e.g. `start()` or `spawn()` -- `run()` is squattered by the deprecated API.